cmp/layout/Frame.js

```diff
@@ -7,21 +7,23 @@
 import {Component} from 'react';
 import {elemFactory, HoistComponent, LayoutSupport} from '@xh/hoist/core';
 
+import './Layout.scss';
 import {box} from './Box';
 
 /**
- * A Box class that flexes to grow and stretch within its *own* parent via flex:'auto', useful for
- * creating nested layouts.
+ * A {@see Box} class that flexes to grow and stretch within its *own* parent via `flex:'auto'`.
+ * Useful for creating space-filling nested layouts.
  *
  * VFrame and HFrame variants support internal vertical (column) and horizontal (row) flex layouts.
  */
 @HoistComponent
 @LayoutSupport
 export class Frame extends Component {
+    baseClassName = 'xh-frame';
     render() {
         return box({
             ...this.props,
-            flex: 'auto'
+            className: this.getClassName()
         });
     }
 }
```

Comment:
so according to the logic of this change isn't Frame just a quiet alias for HFrame?  Shouldn't we just drop it?

Reply:
Yes good catch.  I think by our logic `.xh-frame` should just declare `flex: 1 1 auto;` (as a child we want it to be space-filling) but not `display: flex` (as a parent we don't want to say it must flex).